Classes/Game.ts

```diff
@@ -0,0 +1,101 @@
+import Player from "./Player.js";
+import Computer from "./Computer.js";
+import {
+    btnRestart,
+    container1,
+    container3,
+    description,
+    imgComputer,
+    imgPlayer,
+    roundCounter,
+    ties,
+    winComputer,
+    winPlayer
+} from "../selectors/selectors.js";
+
+class Game {
+
+    tiesScore: number = 0;
+    rounds: number = 1;
+    player: Player;
+    computer: Computer;
+
+    options = {
```

Comment:
Wenn du Typescript nutzt, dann kannst du einen Type GameChoice erstellen:
`type GameChoice = "rock" | "paper" | "scissor";`
Und dann kannst du immer strings statt Nummern verwenden, ist viel einfacher zu lesen :)
In C++ verwendet man Nummern als Micro-Optimierung, das brauchen wir in JS nicht. 

Mehr dazu hier:
https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html#composing-types